content/_index.md

```diff
@@ -3,4 +3,7 @@ layout: page
 title: Startwords
 ---
 
-Read more about the project on the [about page](/about).
+startword, <abbr title="noun">n.</abbr>
+: (computer science) A string of letters that identifies the beginning of
+    a valid sequence in a specified language.
+: (literature) The first appearance of a genre in print.
```

Comment:
Is there a way to use the about page summary here instead of duplicating the content? (probably not high priority, doesn't need to be addressed for this PR)

Reply:
I don't think I know how to select the about page to get its summary here, but that makes sense to me!